vexctl.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: vexctl
+  version: 0.2.3
+  epoch: 0
+  description: A tool to create, transform and attest VEX metadata
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - go
```

Comment:
Just checking: does this actually need go at runtime? Most Go apps don't need Go as a runtime dep, unless maybe they shell out to `go build` or something (like `ko` does).

This is as much an admission of my inexperience about vexctl as anything else.